evm/auxiliary/user_account_contract/transaction.py

```diff
@@ -71,18 +61,98 @@ def __init__(
         min_block,
         max_block,
         gas_price,
-        msg_data
+        msg_data,
+        v,
+        r,
+        s,
     ):
+        self.destination = destination
+        self.value = value
+        self.min_block = min_block
+        self.max_block = max_block
+        self.nonce = nonce
+        self.int_gas_price = gas_price
+        self.msg_data = msg_data
+
+        self.v = v
+        self.r = r
+        self.s = s
+
+        data = assemble_data_field(self, include_signature=True)
+
         super().__init__(
             chain_id=chain_id,
             shard_id=shard_id,
             to=to,
-            data=EMPTY_DATA,
+            data=data,
             gas=gas,
             gas_price=gas_price,
             access_list=access_list,
             code=b'',
         )
+
+    def create_unsigned_transaction(self):
```

Comment:
To stick with the existing transaction API this should be a `classmethod`.  This API exists to reduce the cumbersome import of both the signed and unsigned classes, allowing you to create unsigned transactions from the main transaction class.  I'd be fine with this just being an `*args, **kwargs` passthrough to reduce size since there are 10+ arguments.